helm/tenant/values.yaml

```diff
@@ -61,6 +61,15 @@ tenant:
       topologySpreadConstraints: [ ]
       ## Configure Runtime Class
       # runtimeClassName: ""
+      securityContext:
```

Comment:
`securityContext` is already defined above as empty braces:

```yaml
      ## Configure security context
      securityContext: { }
```

Shouldn't we keep only one copy of this value?